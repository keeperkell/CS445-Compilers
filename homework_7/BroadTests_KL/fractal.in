u
a 100000000
o 10000
g
x
