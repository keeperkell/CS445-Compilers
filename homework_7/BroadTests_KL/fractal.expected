WARNING(120): character is 4 characters long and not a single character: '' by ''.  The first char will be used.
WARNING(140): Variable 'JuliaVals' may be uninitialized when used here.
Loading file: BroadTests/fractal.tm
. . . . . . . . .                               . . . . . . . . . . . .
. . . . . . .                                         . . . . . . . . . .
. . . . .                         ; ; ; ; # - - ; ;       . . . . . . . .
. . . .                         ; ; ; ; - - I W + - ; ; ;       . . . . . .
. . . .                         ; ; ; ; ; - - I @ @ O - ; ; ;       . . . . .
. . .                         ; ; ; ; ; - - I O @ @ I + - ; ; ;       . . . .
. . .                       ; ; ; ; ; ; - - + @ @ @ @ @ I - - ; ; ;       . . .
. .                       ; ; ; ; ; - - + + I # @ @ @ @ I + - - - ;         . .
. .                       ; ; ; ; ; - + I I I V # @ @ @ @ V I + + + - ;         .
.                       ; ; ; - - - + @ @ W @ @ @ @ @ @ @ @ @ V O V I - ;
.                     ; ; - - - - + I W @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ - ;
; ; - - - - - + + O # @ @ @ @ @ @ @ @ @ @ @ @ @ @ O - ;
.               ; ; + + + - - + + I O @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ O + - ;
; ; ; - + # O I O O I I O V @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ - ;
; ; ; ; - - + W @ # # @ @ V V @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ V - ;
; ; ; ; ; - - - I V @ @ @ @ @ @ # @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ I - ; ;
; ; ; ; ; - - - I O @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ I - ; ;
; ; ; ; - + + I @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ # + - ; ;
. - + I I I I O W @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ I - - ; ;
I O W # # # # @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ W I - - ; ;
- - ; + + + I O # @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ O + - ; ;
; ; ; ; - - + I @ V @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ + - ; ;
; ; ; ; - - - + I W @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ I - ; ;
; ; ; ; - - - I O @ @ @ @ @ @ W @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ I - ;
; ; ; ; - - + @ V V W # V O O # @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ - ;
.           ; ; ; + @ I + I I I I I O @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ V @ - ;
.                 ; - - - - - - + + I @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ @ I + - ;
; - - - - - + + O W @ @ @ @ @ @ @ @ @ @ @ @ @ @ V - ;
.                     ; ; ; - - - + I @ @ @ @ @ @ @ @ @ @ @ @ # @ @ @ - ;
. .                     ; ; ; ; - - + @ @ V W @ @ @ @ @ @ @ @ O I O I ;
.                         ; ; ; ; - - + + I O @ @ @ @ @ O I + + - ; ;         .
. .                         ; ; ; ; ; - - + I W @ @ @ @ I - - - ; ;         . .
. .                         ; ; ; ; ; - - + O # @ @ V + - - ; ; ;       . . .
. . .                         ; ; ; ; ; - - + O # # I - ; ; ; ;       . . . .
. . .                         ; ; ; ; ; - - I @ O @ ; ; ; ;       . . . . .
. . . . .                         ; ; ; ; - O I + - ; ;       . . . . . . .
. . . . .                           ; ; ; - - ; ;         . . . . . . . .
. . . . . . .                                       . . . . . . . . . .
. . . . . . . . . . . . . . . . . . . . . .
. . . . . . . . . . . . . . . . . . . . . .       ; ; ; ; ; ; ; ;
. . . . . . . . . . . . . . . . . . . . .     ; ; - T + - ; ; ; ; ; ; ; ;
. . . . . . . . . . . . . . . . . . . .       ; ; - I T + - - ; ; ; ; ; ; ;
. . . . . . . . . . . . . . . . . . . .       ; ; ; + V W I - - - ; ; ; ; ; -
. . . . . . . . . . . . . . . . . . .         ; ; ; - O T O I + - - - - - - +
. . . . . . . . . . . . . . . . . . .         ; ; ; - - + F T W V + + + + + I T
. . . . . . . . . . . . . . . . . .           ; ; ; - - + I V T O I I + + I W T
. . . . . . . . . . . . . . . . . .             ; ; ; - - + I O T F O I I O W T V
. . . . . . . . . . . . . . . . .               ; ; - - - + I O F T W W T W T F I
. . . . . . . . . . . . . . . .               ; ; ; - - + + O W T T T T W T V I +
. . . . . . . . . . . . . . .                 ; ; - - + + I V T W V V O O V W + -
. . . . . . . . . . . . . . .                   ; ; - I I I O W T O O I I I I + - -
. . . . . . . . . . . . . .                     ; ; I F V V V T F O I + + + + - - ;
. . . . . . . . . . . . .                     ; ; - + O F T T F T I + + + - - - ; ;
. . . . . . . . . . . .                       ; ; - + I V T F O O + + + - - - ; ; ;
. . . . . . . . . . . .                     ; ; ; - + I O T W O I + + - - - ; ; ; ;
. . . . . . . . . . .                       ; ; - - + I T T W I I + - - - - ; ; ;
. . . . . . . . . . .                       ; ; - - - + I V T T I + - - - - ; ; ;
. . . . . . . . . . .                     ; ; ; - - - + I W T T O + - - - ; ; ;
. . . . . . . . .                     ; ; ; - - - - + I F T T O + - - - ; ;
. . . . . . . . .                 ; ; ; ; - - - - + I I V T W O + - - ; ;
. . . . . . . .                 ; ; ; ; - - - - + + I O V T O I + - - ; ;
. . . . . . . .             ; ; ; ; ; ; - - - + + + I V F T V I + - ; ;
. . . . . . . .           ; ; ; ; ; ; - - + + + + I V T T F T T I - ; ;
. . . . . . .         ; ; ; ; ; ; ; - - + + + + I O F T V O I O I ; ;
. . . . . . .       ; ; ; ; ; ; ; - - I O I I I O V T F O I + + - ; ;
. . . . . .       ; ; ; ; - - - + + O T V V T V W T F I + + - - ; ;
. . . . . .     ; - - - - - + + + O V T T T T T T V O + + - - ; ; ;
. . . . . .     - O I + + I I I O F T V W V V F F O I + - - - ; ; ;             .
. . . . .     - W T T T O O V V T V O I I I O T V I + - - ; ; ;             . .
. . . . . .   ; + + I O T T F T O I + + + I O T F O + - - ; ; ;           . . .
. . . . .   ; ; - - I T O I W I + + + + + + O W T V - - ; ; ;         . . . .
. . . . .     ; ; - + F + + - - - ; - - - - + I W O + ; ; ;         . . . . .
. . . . .     ; ; O I - - ; ; ; ; ; ; - - - + V T T ; ; ;       . . . . . .
. . . . .         ; ; ; ; ; ; ; ; ; ; ; ; - + O + - ; ;     . . . . . . . .
. . . . .                   ; ; ; ; ; ; ; - T - ;       . . . . . . . . .
. . . . .                         ; ; ; ; ;         . . . . . . . . . .
Bye.
